Ciac 4934 jira service management (#34860)
demisto/content: Packs/AtlassianJiraServiceManagement/Integrations/AtlassianJiraServiceManagement/AtlassianJiraServiceManagement_description.md

### Create your API token

> To create your API token, follow the instructions in [this atlassian support page](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html)

Reviewer: ```suggestion
To create your API token, follow the instructions in [this Atlassian support page](https://confluence.atlassian.com/enterprise/using-personal-access-tokens-1026032365.html)
```

> **Please note:** The owner of the token has to be a minimum "Asset Developer". In addition to the required permissions, the user must have a premium license to Jira Service Management.

Reviewer: ```suggestion
**Note:** The owner of the token has to be at a minimum an "Asset Developer". In addition to the required permissions, the user must have a premium license to Jira Service Management.
```

### Version support:

> This pack currently supports only on-prem clients with JSM version 5.4.15  

Reviewer: ```suggestion
This pack currently supports only on-prem clients with JSM version 5.4.15.  
```
